x/orderbook/keeper/bet_place.go

```diff
@@ -63,30 +64,32 @@ func (k Keeper) ProcessBetPlacement(
 	if err := k.fund(types.OrderBookLiquidityFunder{}, ctx, bettorAddress, fInfo.fulfilledBetAmount); err != nil {
 		return nil, err
 	}
-	_ = k.PublishOrderBookEvent(ctx, bookUID)
+	k.PublishOrderBookEvent(ctx, bookUID)
 	return fInfo.fulfillments, nil
 }
 
-func (k Keeper) PublishOrderBookEvent(ctx sdk.Context, orderBookUID string) error {
+func (k Keeper) PublishOrderBookEvent(ctx sdk.Context, orderBookUID string) {
 	event := types.NewOrderBookEvent()
 	boes, err := k.GetOddsExposuresByOrderBook(ctx, orderBookUID)
 	if err != nil {
-		return err
+		log.Println(err)
```

Comment:
Keeper has built-in logger
Similar to this
`k.Logger(ctx).Error(fmt.Sprintf("end block number %d error: %s", ctx.BlockHeight(), err))`